qiskit/mapper/_layout.py

```diff
@@ -150,6 +150,23 @@ def swap(self, left, right):
         self[left] = self[right]
         self[right] = temp
 
+    def combine_into_edge_map(self, another_layout):
+        """ Combines self and another_layout into an "edge map".
```

Comment:
Can you expand on this docstring to better describe what this method does and why it's needed?

Reply:
I added the following explanation in 3f116e3a, do you think that would make it?
```
 Combines self and another_layout into an "edge map". For example

            self       another_layout  resulting edge map
         qr_1 -> 0        0 <- q_2         qr_1 -> q_2
         qr_2 -> 2        2 <- q_1         qr_2 -> q_1
         qr_3 -> 3        3 <- q_0         qr_3 -> q_0

         The edge map is used to compose dags via, for example, compose_back.
```

Reply:
cool yeah now I understand.